quickwit/quickwit-config/src/index_config/serialize.rs

```diff
@@ -87,6 +87,13 @@ impl IndexConfigForSerialization {
 
         let index_uri = self.index_uri_or_fallback_to_default(default_index_root_uri)?;
 
+        if index_uri.protocol().is_file() {
+            warn!(
+                "index_uri = {}. Using file-backed index comes with certain limitations, please \
```

Comment:
Same as below

Reply:
How about we give the link as https://quickwit.io/docs/configuration/index-config ?